index.html

```diff
@@ -6,9 +6,9 @@
 
     <link rel="shortcut icon" href="favicon.ico">
 
-    <link type="text/css" rel="stylesheet" href="../dhis-web-commons/material-design-icons/material-icons.css">
-    <link type="text/css" rel="stylesheet" href="../dhis-web-commons/fonts/roboto.css">
-    <link type="text/css" rel="stylesheet" href="../dhis-web-commons/font-awesome/css/font-awesome.min.css">
+    <link type="text/css" rel="stylesheet" href="/dhis-web-commons/material-design-icons/material-icons.css">
+    <link type="text/css" rel="stylesheet" href="/dhis-web-commons/fonts/roboto.css">
+    <link type="text/css" rel="stylesheet" href="/dhis-web-commons/font-awesome/css/font-awesome.min.css">
 
```

Comment:
Ok, other than that, `"href": "*"` is the way to go. I've tested and worked also on my server.